docs/data_onboarding/cloud_tenancies/o365_azure_ad.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: O365 Azure Active Directory
+permalink: /data_onboarding/cloud_tenancies/o365_azure_ad/
+nav_order: 5
```

Comment:
Do we have to change the nav order for pages after this?

Reply:
It's the latest number but i think we have to add as per the order of file hence updating it,